src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -659,15 +659,17 @@ export const Deposit: React.FC<DepositProps> = ({
       }
     },
     [
-      accountId,
-      asset,
-      assetId,
+      asset.chainId,
+      asset.precision,
```

Comment:
side note, have always wondered how useful this syntax is vs just `asset`?

Reply:
`asset` would assume you are using `asset` directly vs `asset.precision` - lint will automagically use the right ones